nvmw.bat

```diff
@@ -75,27 +81,44 @@ exit /b 0
 :install
 setlocal
 
+set NODE_TYPE=node
 set NODE_VERSION=%1
+
+if %NODE_VERSION:~4,1% == / (
+  for /f "tokens=1,2,* delims=/" %%a in ("%NODE_VERSION%") do (
+    set NODE_TYPE=%%a
+    set NODE_VERSION=%%b
+  )
+)
+
 if not %NODE_VERSION:~0,1% == v (

```

Comment:
@fengmk2 shouldnt this be `if not %NODE_VERSION:~0,1% == v if not %NODE_VERSION:~0,1% == l` . if there is a numeric condition that would be better. Otherwise works awesome!